### PR TITLE
Add load case report docs

### PR DESCRIPTION
**Pull request by northwestwitch**
Adds instructions in admin docs on how to load case reports via the command line

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, log out from `cg-vm1` and log in again in the `hasta` server, repeat the `hasta` and `paxa` procedure, which will release the allocated resource (scout-stage) to be used for testing by other users.
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN
- [x] tests executed by CR

### CODE REVIEW

File: docs/admin-guide/loading-case.md [hunk 1]
```diff
@@ -86,9 +64,42 @@ human_genome_build: 37
 
 ```
 
+#### Adding custom reports to the case
+A number of case-associated reports (supported formats: HTML, PDF, Excel) can be loaded and displayed/downloaded on the case sidebar page:
+
+<img width="227" alt="image" src="https://user-images.githubusercontent.com/28093618/201290117-33b1ea53-eb8e-4e80-a5df-edba8b6595fe.png">
+
+While case General report and mtDNA report (the latter only available for non-cancer cases) are generated the moment a user clicks on their link, other types of reports are pre-existing documents that can be associated with the case the moment the case is loaded in Scout (by adding them to the case load config file) or in a second moment, using the command line.
```
Comment (dnil) [inline docs/admin-guide/loading-case.md:72]:
```suggestion
While case General report and mtDNA report (the latter only available for non-cancer cases) are generated the moment a user clicks on their link, other types of reports are pre-existing documents that can be associated with the case immediately when it is loaded in Scout (by adding them to the case load config file) or later, using the command line.
```

File: docs/admin-guide/loading-case.md [hunk 2]
```diff
@@ -86,9 +64,42 @@ human_genome_build: 37
...
+Options:
+  -t, --report-type [delivery|cnv|cov_qc|multiqc|multiqc_rna|gene_fusion|gene_fusion_research]
+                                  Type of report  [required]
+
+```
+
 #### Adding custom images to a case
 
-Scout can display custom images as new panels on the case view or variant view which could be used to display analysis results from a seperate pipeline. The custom images are defined in the case config file and stored in the database. Scout currently supports     `gif`, `jpeg`, `png` and `svg` images.
+Scout can display custom images as new panels on the case view or variant view which could be used to display analysis results from a separate pipeline. The custom images are defined in the case config file and stored in the database. Scout currently supports     `gif`, `jpeg`, `png` and `svg` images.
```
Comment (dnil) [inline docs/admin-guide/loading-case.md:102]: 🔍 💯

File: mkdocs.yml [hunk 3]
```diff
@@ -51,10 +51,12 @@ nav:
           - Dockerfile and docker-compose: 'admin-guide/containers/container-deploy.md'
           - Deploying using Kubernetes: 'admin-guide/containers/kubernetes.md'
           - Deploying as a systemd service with podman: 'admin-guide/containers/systemd.md'
-      - Loading: 'admin-guide/loading.md'
-      - Loading variants: 'admin-guide/loading-variants.md'
+      - Creating a new user: 'admin-guide/loading-users.md'
```
Comment (dnil) [inline mkdocs.yml:54]: 👍

**Review by dnil — APPROVED** [review]
Nice! Perhaps add multiqc_rna etc to the load-config.md as well? We will never be able to keep the latter complete - it needs to be in like tests for that - but why not when you still have it fresh! 😄

**Comment by northwestwitch** [comment]
> Nice! Perhaps add multiqc_rna etc to the load-config.md as well? We will never be able to keep the latter complete - it needs to be in like tests for that - but why not when you still have it fresh!

Good idea, done!

**Pull request merged**